src/WebJobs.Script.WebHost/WebJobsApplicationBuilderExtension.cs

```diff
@@ -116,5 +122,38 @@ public static IApplicationBuilder UseWebJobsScriptHost(this IApplicationBuilder
 
             return builder;
         }
+
+        private static void UseHealthChecks(IApplicationBuilder app)
```

Comment:
Any reason why you don't make this an extension method so you can builder.UseHealthChecks abouve?